### ISSUE TITLE
Tensorflow Cifar-10 evaluation error: Enqueue operation was cancelled

### ISSUE THREAD

**Issue by BlackyTheBlueCat** [tensorflow/tensorflow#1360]
Operating System: Mac Os X 10.9.5

Current tensorflow version 0.7.1

I am trying to get the Cifar-10 tutorial running on my own data set (so using other images, but the same code). Training works fine. However, the evaluation seems to have a problem. Whenever I run the evaluation, I get the following message:

W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fe504a20 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fe403eb0 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fe206b10 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fbe16130 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6f8e6bd80 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fbcca1e0 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fbbb5e30 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6f8daee90 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fe304d50 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fbcb9570 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6f8e01660 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fba2a1a0 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6f8e357a0 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fe207e90 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fe2089b0 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6fa6bfe20 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: shuffle_batch/random_shuffle_queue_enqueue = QueueEnqueue[Tcomp                                                                                                                     onents=[DT_FLOAT, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/ta                                                                                                                     sk:0/cpu:0"](shuffle_batch/random_shuffle_queue, Div, Cast)]]
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt
W tensorflow/core/common_runtime/executor.cc:1102] 0x7fc6f8e689f0 Compute status                                                                                                                     : Cancelled: Enqueue operation was cancelled
         [[Node: input_producer/input_producer_EnqueueMany = QueueEnqueueMany[Tc                                                                                                                     omponents=[DT_STRING], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/c                                                                                                                     pu:0"](input_producer, input_producer/RandomShuffle)]]
I tensorflow/core/kernels/queue_base.cc:286] Skipping cancelled enqueue attempt

Right before this message, if I let the code run long enough, I get the output for a precision (precision @ 1 = xxx). However, the message still irritates me, I looks like an error message...

I've googled this error, and so far I've only found some posts talking about that one should first initialize placeholders before starting the queuerunners. However, since there are no placeholders used anywhere in the code before, and I am very new to tensorflow/machine learning in general, I have no idea where one would put these placeholders, or what they would be.

**Comment by vrv** [comment]
Duplicate of #1341, essentially.

I think this is superfluous logging.  I tried to get rid of this but apparently these logs are useful for debugging deadlocks. @jmchen-g 

Maybe we should just write to log files instead of spamming users.
